scenario/mocking.py

```diff
@@ -631,6 +632,13 @@ def resource_get(self, resource_name: str) -> str:
                 f"resource {resource_name} not found in State. please pass it.",
             )
 
+    def credential_get(self) -> CloudSpec:
+        if not self._state.cloud_spec:
+            raise ModelError(
+                "ERROR cloud spec is empty, initialise it with `scenario.State(cloud_spec=...)`",
```

Comment:
To avoid giving the impression you're allowed to pass in ops.CloudSpec...

```suggestion
                "ERROR cloud spec is empty, initialise it with `scenario.State(cloud_spec=scenario.state.CloudSpec(...))`",
```